core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/sklearn/SklearnBaggingOpDesc.scala

```diff
@@ -0,0 +1,6 @@
+package edu.uci.ics.texera.workflow.operators.sklearn
+
+class SklearnBaggingOpDesc extends SklearnMLOpDesc {
+  model = "from sklearn.ensemble import BaggingClassifier"
```

Comment:
add sklearn to `operator-requirements.txt`?

Reply:
Added.